test/plugin_functional/plugins/kbn_tp_embeddable_explorer/public/app/_index.scss

```diff
@@ -0,0 +1,2 @@
+// Temporary hacks
+@import './hacks';
```

Comment:
Why is there SASS files in the test folder?

Reply:
So this is how it looks without these hacks
<img width="1483" alt="Screen Shot 2019-06-05 at 12 52 23 PM" src="https://user-images.githubusercontent.com/16563603/58974689-00816b00-8791-11e9-9539-5f6826897ad6.png">
<img width="1044" alt="Screen Shot 2019-06-05 at 12 52 26 PM" src="https://user-images.githubusercontent.com/16563603/58974690-00816b00-8791-11e9-97bc-557fb4b37059.png">

This is how it looks with the hacks:

<img width="603" alt="Screen Shot 2019-06-05 at 12 53 27 PM" src="https://user-images.githubusercontent.com/16563603/58974708-0c6d2d00-8791-11e9-8a8e-4bd7a03df40c.png">
<img width="1071" alt="Screen Shot 2019-06-05 at 12 53 33 PM" src="https://user-images.githubusercontent.com/16563603/58974709-0c6d2d00-8791-11e9-957f-93439ee88497.png">

I used code similar to what dashboard does to get that full screen expansion.  This actually caused ci failures so i commented it out but wanted to go back to it after I got at least one green ci.  But I'm thinking now the easiest way to do this, since it's just a test file, is to inline the css.  So I can do that, and also not encounter the ci issue.

Reply:
So when I run the explorer, my view looks like the first screenshot you have. Should the hack be always applied not just in tests since now the test UI is actually different from the rendered UI?

Reply:
Yea that's because I had the line that imported the sass commented out due to ci errors.  But if you download the latest version, after my last commit... then do you still see the first version or the second?

>  Should the hack be always applied not just in tests since now the test UI is actually different from the rendered UI?

Not quite sure I follow this.  I think the extra styling should always be applied if you are using this plugin, whether for exploration or in ci being functionally tested. But I might be misunderstanding your question.

Reply:
Yep, ok I see the correct version now (full height panel). Thx